scripts/build_packages/upstart/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:14.04
+
+ENV BASE_PATH=/go/src/github.com/grafana/metrictank
+
+ARG PKG
+ARG VERSION
+ARG ARCH
+COPY . $BASE_PATH/
+RUN apt-get update && apt-get install -y curl build-essential libssl-dev libsasl2-dev git bash python pkg-config ruby ruby-dev
+RUN gem install --no-ri --no-rdoc fpm
+RUN $BASE_PATH/scripts/get_go.sh
```

Comment:
not a big fan of this script.  we can probably use the debian images that have go, but i'm not sure if there's a solution for the centos images :'(

Reply:
we could indeed use the debian images with go, i just figured that for consistency it might be nicer to have exactly the same go runtime everywhere.